it/src/main/java/org/corfudb/universe/node/server/CorfuServer.java

```diff
@@ -15,12 +17,20 @@
     CorfuServerParams getParams();
 
     /**
-     * Disconnect a CorfuServer from the network
+     * Disconnect a CorfuServer from the cluster, which creates a complete partition.
      *
      * @throws NodeException thrown in case of unsuccessful disconnect.
      */
     void disconnect();
 
+    /**
+     * Disconnect a CorfuServer from a list of other servers, which creates a partial partition.
```

Comment:
Can you mention that this is a symmetric disconnect?

Reply:
Done.